RazorGenerator.Core/CodeLanguageUtil.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace RazorGenerator.Core
+{
+    abstract public class CodeLanguageUtil
+    {
+
+        #region "Language Util Factory"
+        public static CodeLanguageUtil GetLanguageUtilFromFileExtension(string extension)
+        {
+            if (extension.StartsWith("."))
+            {
+                extension = extension.Substring(1);
+            }
+            extension = extension.ToLower();
```

Comment:
It is your code base but personally I prefer the neatness of the switch statement.  If it was me, I'd prefer to sacrifice a few potential CPU cycles for that?